apis/sgv2-restapi/pom.xml

```diff
@@ -12,6 +12,7 @@
     <failsafe.useModulePath>false</failsafe.useModulePath>
     <quarkus.container-image.group>stargateio</quarkus.container-image.group>
     <quarkus.container-image.name>restapi</quarkus.container-image.name>
+    <quarkus.container-image.tag>v${project.version}</quarkus.container-image.tag>
```

Comment:
Needed so the tag is `v2.0.0-BETA-2-SNAPSHOT` and not just `2.0.0-BETA-2-SNAPSHOT` (copied from `sgv2-graphqlapi/pom.xml`)